app/controllers/hq/partners_controller.rb

```diff
@@ -5,4 +5,48 @@ class Hq::PartnersController < ApplicationController
   def index
     @partners= Partner.all
   end
+
+  def edit
+    load_partner
+    load_associations
+  end
+
+  def update
+    load_partner
+    load_associations
+    build_partner
+    save_partner or render 'edit'
+  end
+
+private
+  def load_partner
```

Comment:
I had actually considered it but, to me, assigns is about variables, where loads brings to attention stuff is being brought in from the database.
I'm comfortable with either.